yucca/umbrel-app.yml

```diff
@@ -0,0 +1,29 @@
+manifestVersion: 1.1
+id: yucca
+category: automation
+name: Yucca
+version: "0.10.0"
+tagline: Simple NVR solution video surveillance
+description: >-
+  Simple NVR solution video surveillance for home and business.  
+  Perfect for both a solution for safe cities with thousands of cameras and launching a subscriber service, as well as for home use self-hosted.
+
+releaseNotes: >-
+  Init Yucca on Umbrel
+
+developer:  Yucca Stream
+website: https://yucca.app/
+repo: ""
```

Comment:
You can add the url to your github repo for Yucca here. If you don't, then the source code will show as "Private" in the app store UI.

Reply:
Yes, this Private